src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -395,6 +407,16 @@ protected function prepare_item_for_database( $request ) {
 			$changes->post_excerpt = $template->description;
 		}
 
+		if ( 'wp_template_part' === $this->post_type ) {
+			if ( isset( $request['area'] ) ) {
+				$changes->tax_input['wp_template_part_area'] = filter_template_part_area( $request['area'] );
```

Comment:
I guess this and below call should be `_filter_block_template_part_area`, no?

Reply:
Good catch, I updated them.